frontend/src/store/modules/v1/projectIamPolicy.ts

```diff
@@ -132,151 +131,81 @@ export const useProjectIamPolicy = (project: MaybeRef<string>) => {
   return { policy, ready };
 };
 
-export const useCurrentUserIamPolicy = () => {
-  const iamPolicyStore = useProjectIamPolicyStore();
-  const projectStore = useProjectV1Store();
-  const currentUser = useCurrentUserV1();
-
-  watchEffect(() => {
-    // Fetch all project iam policies.
-    Promise.all(
-      projectStore.projectList.map((project) =>
-        iamPolicyStore.getOrFetchProjectIamPolicy(project.name)
-      )
-    );
-  });
-
-  // hasWorkspaceSuperPrivilege checks whether the current user has the super privilege to access all databases. AKA. Owners and DBAs
-  const hasWorkspaceSuperPrivilege = computed(() =>
-    hasWorkspacePermissionV2("bb.projects.list")
-  );
+const checkProjectIAMPolicyWithExpr = (
+  user: User,
+  project: ComposedProject,
+  role: string,
+  bindingExprCheck: (expr: Expr) => boolean
+): boolean => {
+  const roleList = usePermissionStore().currentRoleListInProjectV1(project);
```

Comment:
`currentRoleListInProjectV1` might contain expired roles. We can update the `roleListInIAM` to fix it later in another PR

Reply:
How about checking it with  https://github.com/bytebase/bytebase/pull/13725/commits/25e6565bdcc5c1e503be8cfacc0b69f2f022e565